Makefile

```diff
@@ -58,9 +58,9 @@ upgrade: ## update the requirements/*.txt files with the latest packages satisfy
 	pip-compile --upgrade -o requirements/quality.txt requirements/quality.in
 	pip-compile --upgrade -o requirements/travis.txt requirements/travis.in
 	pip-compile --upgrade -o requirements/dev.txt requirements/dev.in
-	# Let tox control the Django version for tests
-	sed '/^[d|D]jango==/d' requirements/test.txt > requirements/test.tmp
-	mv requirements/test.tmp requirements/test.txt
+	# Let tox control the Django and djangorestframework versions for tests
+	sed -i '/^[d|D]jango==/d' requirements/test.txt
```

Comment:
Don't you need an empty-string argument for -i  ?

Reply:
@nedbat good catch, on linux the argument is optional, but on Macs it is required.  I've updated the PR to work with both.